docs/asciidoc/modules/ROOT/pages/deprecations-and-additions/index.adoc

```diff
@@ -2,6 +2,12 @@
 = Deprecations and additions
 :description: This chapter lists all the features that have been removed, deprecated, added or extended in the recent versions of APOC.
 
+[WARNING]
+====
+This is the page for APOC Extended documentation.
+For APOC Core, go to the https://neo4j.com/docs/apoc/{branch}/deprecations-and-additions/[APOC Core page].
+====
```

Comment:
Maybe include something in the message saying apoc extended is not officially supported by Neo4j?

Reply:
Would this work perhaps?

This is the page for APOC Extended documentation.
For the officially supported APOC Core, go to the https://neo4j.com/docs/apoc/{branch}/deprecations-and-additions/[APOC Core page].